code/modules/cargo/packs/engineering.dm

```diff
@@ -121,18 +117,6 @@
 	crate_name = "power cell crate"
 	crate_type = /obj/structure/closet/crate/engineering/electrical
 
-
-/datum/supply_pack/engineering/siezedpower
```

Comment:
these have a reason to be here, its for electroplating

Reply:
electro-what?

Reply:
Electroplating, plating tools in bronze. Thats why this crate is here.

Reply:
You need no damn high plus power cell to make stupid bronze tools as far I can tell by looking at the current crafting recipes for them.